src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -88,7 +88,7 @@ private enum Mode
             /// <summary>
             /// Just adding a file version for the generated urls.
             /// </summary>
-            FileVersion = 0,
+            AppendVersion = 0,
             /// <summary>
```

Comment:
While here please add blank lines within this `enum` to separate the members. Same in `ScriptTagHelper`.